### DISCUSSION TITLE
Connecting Aubess devices

### DISCUSSION THREAD

**Discussion by Minemat7** [jasonacox/tinytuya#318] [Q&A]
Hi, I just bought Aubess smart plugs to use with tuya. I can use them in smart life app, but when I try to search them using the tinytuya scan command, the program doesn't find them. Please, is there any way to use those aubess plugs in tinytuya?
Thanks

**Comment by jasonacox** [comment]
Hi @Minemat7 - 

SmartLife app can use the Cloud to find your device. TinyTuya looks at your local network. Tuya devices emit a UDP packet for discovery that TinyTuya listens for when you do the "scan" or "wizard" function.  Some things to consider:

* When you have Smartlife app open (or other service conneciting to the devices) some of the devices stop emitting UDP packets - close the app when you scan
* Some devices get into a bad state where they only send updates to the Cloud (which Smartlife will read)
* If your devices are not on the same network with the host running TinyTuya, it will not see them.  You can attempt to scan for them using the force option: 

```bash
python3 -m tinytuya scan -f      
```

**Reply by Minemat7** [reply]
I tried closing the app and forced scanning, but it still doesn't find them. Is there any other way to get their address, Device ID and Local Key?

**Reply by jasonacox** [reply]
Interesting. If you don't find them via the force scan, that would seem to indicate that they are not on your network. About the only other way to find the devices would be if your router shows devices on your network and could provide the IP address information.

You can still get the Device IDs and Keys using the wizard (see the [instructions](https://github.com/jasonacox/tinytuya#setup-wizard---getting-local-keys)):

```
python3 -m tinytuya wizard
```

**Reply by Minemat7** [reply]
I finally managed to add the plugs and get their keys, but now, when I use them in code it acts like the plugs don't exist and it just ignores them

**Reply by jasonacox** [reply]
I suggest turning on debug mode to see what is happening.
